components/button/demo/disabled.md

```diff
@@ -1,32 +1,35 @@
 ---
 order: 3
-title: 不可用
+title:
+  zh-CN: 不可用状态
+  en-US: Disabled
 ---
 
+## zh-CN
+
 添加 `disabled` 属性即可让按钮处于不可用状态，同时按钮样式也会改变。
 
+## en-US
+
+To mark a button as disabled, add the `disabled` property to the `Button`.
```

Comment:
IMO, we should use property to follow the convention of [React](http://facebook.github.io/react/docs/transferring-props.html).